airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/AirbyteType.java

```diff
@@ -168,12 +176,16 @@ record OneOf(List<AirbyteType> options) implements AirbyteType {
     public LinkedHashMap<String, AirbyteType> asColumns() {
       final long numObjectOptions = options.stream().filter(o -> o instanceof Struct).count();
       if (numObjectOptions > 1) {
-        throw new IllegalArgumentException("Can't extract columns from a schema with multiple object options");
+        LOGGER.warn("Found multiple object options in a oneOf schema. This is not supported.");
+        return new LinkedHashMap<>();
```

Comment:
Alternatively, we could pull the columns from the first struct? :P but maybe that's more confusing than helpful

Reply:
this situation isn't actually possible right now, so :shrug: we can answer that question whenever we merge UnsupportedOneOf and OneOf